src/AzAcme.Core/Providers/KeyVault/AzureKeyVaultCertificateStore.cs

```diff
@@ -50,7 +50,7 @@ public async Task<CertificateCsr> Prepare(CertificateRequest request)
             var op = this.GetExistingOperationOrNull(request.Name);
 
             bool create = false;
-            if (op == null)
+            if (op == null || string.IsNullOrEmpty(request.Subject))

```

Comment:
Why is this change needed? From a glance, it appears that the operation would be null and therefore we should create the request. Can you explain why we need this?

Reply:
I get your question, it doesn't really make sense just looking at the code. 
The reason for adding this was to ensure there was a create action if the subject was blank. 
I was finding that `op` came back as populated and did not prove true in the `if else` block under this if. I suppose there is a chance that things were a little skewed due to the various times I ran this so I can try this without this adjustment.

Reply:
Have reverted this change after confirming it seems fine without it.